includes/REST_API/Link_Controller.php

```diff
@@ -105,17 +105,13 @@ public function parse_link( $request ) {
 
 		$data = get_transient( $cache_key );
 		if ( ! empty( $data ) ) {
-			return json_decode( $data, true );
+			return rest_ensure_response( json_decode( $data, true ) );
```

Comment:
Does this still need `json_decode`? I thought `rest_ensure_response` handled all this.

Reply:
I'll test.